libmachine/state/state.go

```diff
@@ -4,27 +4,27 @@ package state
 type State int
 
 const (
-	None State = iota
+	reserved0 State = iota
 	Running
-	Paused
-	Saved
+	reserved2
+	reserved3
```

Comment:
Yes it will help the upgrade. We dont have to bump everything at the same time.

The RPC server is using int for the state and it would be a problem if None == Running.